### PR TITLE
#45 Remove deprecated and unused dependencies

### PR DESCRIPTION
**Pull request by migo315**

### CODE REVIEW

**Comment by migo315** [comment]
Successfully tested locally. I will handle it as a (deprecation) fix and tag it with 3.1.1 after merge.

File: composer.json [hunk 1]
```diff
@@ -12,8 +12,12 @@
   ],
   "require": {
     "php": "^5.6||^7.0",
-    "symfony/framework-bundle": ">=2.7",
-    "doctrine/common": "^2.3",
+    "symfony/dependency-injection": "^2.7|^3.0|^4.0",
```
Comment (b3nl) [inline composer.json]: Why not https://getcomposer.org/doc/articles/versions.md#version-range ?

Comment (b3nl) [reply composer.json]: "Ranges separated by a space ( ) or comma (,) will be treated as a logical AND."

Comment (migo315) [reply composer.json]: Thats possible but not recommended: "Consider using the caret operator instead for safety". In addition, you will always find in various issues for versioning the recommendation to use the caret. The "big" bundles also use caret. Ranges are not really common. Finally, I find this implementation understandable despite the longer spelling.

Comment (b3nl) [reply composer.json]: So, you are quoting (and your inception starts with) the part for the "unbound range". Who said you should use an unbound range?

Comment (b3nl) [reply composer.json]: (And then I would suggest, that you use the documented style with a double pipe for an OR.)

**Pull request merged**